### PR TITLE
[panels] Support latest changes in other componennts

### PR DESCRIPTION
**Pull request by jgbarah**
Kibiter now supports a title for the dashboard. New method for uploading it.

SortingHat now supports a new flag when initializing a database, to avoid an error message which was not really an error.

### CODE REVIEW

File: mordred/task_panels.py [hunk 1]
```diff
@@ -439,6 +439,10 @@ def __init__(self, conf):
                 raise
         # Get the active data sources
         self.data_sources = self.__get_active_data_sources()
+        if 'shortname' in self.conf['general']:
+            self.project_name = self.conf['general']['short_name']
+        else:
+            self.project_name = 'GrimoireLab'
```
Comment (acs) [inline mordred/task_panels.py]: `GrimoireLab` should be a class constant so it is easier to find it.

**Comment by acs** [comment]
There are the flake8 issue with the new code and an strange error with the tests:

`run_tests.py: error: unrecognized arguments: --reuse`

Could you fix them?

**Comment by jgbarah** [comment]
Flake8 error should be fixed, sorry about that.

The other issue seems to me relate to the version of SortingHat. This works only with the latest sortinghart in master branch, where a new option `--reuse` has been introduced. Since we still don't have a package uploaded to pypi with that version, it fails :-(

**Comment by acs** [comment]
Ok, merging the PR with the hope that the new SH version fixes the error! But in this case, it is critical to have the sortinghat dependency correct or mordred will be broken :(

**Comment by jgbarah** [comment]
Yessssss. It should be on the latest version of SortingHat, once it is released...

**Pull request merged**